tbx/core/blocks.py

```diff
@@ -110,6 +110,22 @@ class ImageBlock(ImageWithAltTextBlock):
     attribution = blocks.CharBlock(required=False)
 
 
+class WideImageBlock(ImageWithAltTextBlock):
+    """
+    An image that is rendered wider in the template than an ImageBlock.
+    In addition to specifying optional alt text for an image, this block allows
+    for specifying a caption, attribution and whether the image is decorative.
+    """
+
+    image_is_decorative = blocks.BooleanBlock(
+        required=False,
+        default=False,
+        help_text="If checked, this will make the alt text empty.",
+    )
+    caption = blocks.CharBlock(required=False)
+    attribution = blocks.CharBlock(required=False)
```

Comment:
**Issue**: This block appears to be exactly the same as the `ImageBlock` above it:

https://github.com/torchbox/torchbox.com/blob/70ffadd129e2fb3b219d5f1c921f586c170c6e8c/tbx/core/blocks.py#L98-L110

**Suggestion**: If the structure of the wide image block is no different from that of the existing image block, then we can resuse the latter, specifying a template and optionally a different label. Please see the suggested code in my comment on the `StoryBlock`.

Reply:
Updated in f40e404a010704cfa57a609f47027afc3e6b7e03